app/code/Magento/CurrencySymbol/Controller/Adminhtml/System/Currency/FetchRates.php

```diff
@@ -37,18 +37,36 @@ public function execute()
             $errors = $importModel->getMessages();
             if (sizeof($errors) > 0) {
                 foreach ($errors as $error) {
-                    $this->messageManager->addWarning($error);
+                    $this->messageManager->addComplexWarningMessage(
+                        'addUnescapedMessage',
+                        [
+                            'text' => $error,
+                        ]
+                    );
                 }
-                $this->messageManager->addWarning(
-                    __('Click "Save" to apply the rates we found.')
+                $this->messageManager->addComplexWarningMessage(
+                    'addUnescapedMessage',
```

Comment:
What is the reason to use unescaped Message here? Seems message should not contain any HTML markup.

Reply:
I'll check this, I can't remember if it was about double quotes or if it is a mistake